django_projects/locallibrary/catalog/templates/catalog/author_confirm_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base_generic.html" %}
+
+{% block content %}
+{% load i18n %}
+<h1>{% trans 'Delete Author' %}</h1>
+
+<p>{% trans 'Are you sure you want to delete the author:' %} {{ author }}?</p>
+
+<form action="" method="POST">
```

Comment:
ko set action thì viết gọn lại em, tương tự cho những file khác nha
```suggestion
<form method="POST">
```